INTEGRATION.md

```diff
@@ -0,0 +1,46 @@
+# ![](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square) Integration  <!-- omit in toc -->
```

Comment:
Needs
- a header  – see https://github.com/ipfs/specs/blob/main/template.md
- short IPIP in `IPIP/` with motivation why we add this – see https://github.com/ipfs/specs/blob/main/IPIP/0000-template.md